### PR TITLE
OSASINFRA-2168: Docs: update OpenStack requirements

### PR DESCRIPTION
**Pull request by gryf**

### CODE REVIEW

File: docs/user/openstack/README.md [hunk 1]
```diff
@@ -69,6 +69,25 @@ In addition, it covers the installation with the default CNI (OVNKubernetes), as
...
+- Keystone
+- Neutron
+- Nova
+  - with Metadata service enabled
+- Placement
+- Glance
+- Storage solution for the image registry:
+  - Swift
+  - Cinder with dynamically-provisioned volumes
+  - Manila for ReadWriteMany (RWX) which is required for HA
```
Comment (mandre) [inline docs/user/openstack/README.md:84]: As far I know, CIRO won't provision the image-registry in a manila volume on day 1, but it's possible to move it there as a day 2 operation. To be double-checked...

File: docs/user/openstack/README.md [hunk 2]
```diff
@@ -69,6 +69,25 @@ In addition, it covers the installation with the default CNI (OVNKubernetes), as
...
+
+- Keystone
+- Neutron
+- Nova
+  - with Metadata service enabled
+- Placement
+- Glance
+- Storage solution for the image registry:
+  - Swift
+  - Cinder with dynamically-provisioned volumes
```
Comment (mandre) [inline docs/user/openstack/README.md:83]: `with dynamically-provisioned volumes` is irrelevant here.

File: docs/user/openstack/README.md [hunk 3]
```diff
@@ -69,6 +69,25 @@ In addition, it covers the installation with the default CNI (OVNKubernetes), as
 
 ## OpenStack Requirements
 
+Before starting to install OpenShift, ensure following OpenStack services are
```
Comment (mandre) [inline docs/user/openstack/README.md:72]: The OpenShift installation on OpenStack platform relies on a number of core services being available.

File: docs/user/openstack/README.md [hunk 4]
```diff
@@ -69,6 +69,25 @@ In addition, it covers the installation with the default CNI (OVNKubernetes), as
...
+available, as there are couple of them which we are relying on:
+
+- Keystone
+- Neutron
+- Nova
+  - with Metadata service enabled
+- Placement
```
[the remark this answers is not in the document]
Comment (stephenfin) [reply docs/user/openstack/README.md:79]: Placement is a separate service. It's a hard requirement of Nova.

[the remark this answers is not in the document]
Comment (mandre) [reply docs/user/openstack/README.md:79]: Exactly my point, there's no reason to mention Placement as a separate service. It's either together with Nova, or not at all.

File: docs/user/openstack/README.md (hunk 3 above)
Comment (gryf) [reply docs/user/openstack/README.md:72]: ack

File: docs/user/openstack/README.md (hunk 1 above)
Comment (gryf) [reply docs/user/openstack/README.md:84]: Perhaps just mentioning Manila would be enough?

File: docs/user/openstack/README.md (hunk 2 above)
Comment (gryf) [reply docs/user/openstack/README.md:83]: ack

File: docs/user/openstack/README.md [hunk 5]
```diff
@@ -69,6 +69,25 @@ In addition, it covers the installation with the default CNI (OVNKubernetes), as
...
+- Glance
+- Storage solution for the image registry:
+  - Swift
+  - Cinder with dynamically-provisioned volumes
+  - Manila for ReadWriteMany (RWX) which is required for HA
+- Nova metadata
```
[the remark this answers is not in the document]
Comment (gryf) [reply docs/user/openstack/README.md:85]: Aww, my mistake. Will remove.

File: docs/user/openstack/README.md [hunk 6]
```diff
@@ -69,6 +69,25 @@ In addition, it covers the installation with the default CNI (OVNKubernetes), as
...
+
+Optionally there might be required:
+
+- Octavia in case of [Kuryr networking](kuryr.md)
```
[the remark this answers is not in the document]
Comment (gryf) [reply docs/user/openstack/README.md:89]: Ack

File: docs/user/openstack/README.md [hunk 7]
```diff
@@ -69,6 +69,27 @@ In addition, it covers the installation with the default CNI (OVNKubernetes), as
...
 ## OpenStack Requirements
 
+The OpenShift installation on OpenStack platform relies on a number of core
+services being available:
+
+- Keystone
+- Neutron
+- Nova
+  - with Metadata service enabled
+  - Placement, as it is required by Nova
```
Comment (mandre) [inline docs/user/openstack/README.md:79]: Honestly, I would just remove Placement as we're not directly making calls to this service (it just happens to be a hard requirement for Nova).

File: docs/user/openstack/README.md [hunk 8]
```diff
@@ -69,6 +69,27 @@ In addition, it covers the installation with the default CNI (OVNKubernetes), as
...
+- Glance
+- Storage solution for the image registry, one of:
+  - Swift
+  - Cinder
+  - Manila
+
+Optionally there might be required:
+
+- Octavia
```
Comment (mandre) [inline docs/user/openstack/README.md:88]: Redundant with the sentence right below, please remove.
```suggestion
```

File: docs/user/openstack/README.md [hunk 9]
```diff
@@ -69,6 +69,27 @@ In addition, it covers the installation with the default CNI (OVNKubernetes), as
...
+- Keystone
+- Neutron
+- Nova
+  - with Metadata service enabled
+  - Placement, as it is required by Nova
+- Glance
+- Storage solution for the image registry, one of:
+  - Swift
+  - Cinder
+  - Manila
```
Comment (mandre) [inline docs/user/openstack/README.md:84]: Manila is _not_ a requirement, or option for the image registry at install time. I'd remove it from this list.

**Review by mandre — COMMENTED** [review]
/approve
@stephenfin, what do you think about how the list looks now?

**Review by stephenfin — COMMENTED** [review]
/approve

**Review by mandre — COMMENTED** [review]
/lgtm

**Review by mandre — COMMENTED** [review]
Hmm, what has changed since the last time? :thinking: 
/lgtm

**Comment by gryf** [comment]
I just rebased the patch.

**Pull request merged**